native/src/seal/c/encryptor.cpp

```diff
@@ -131,6 +131,44 @@ SEAL_C_FUNC Encryptor_Encrypt(void *thisptr, void *plaintext, void *destination,
     }
 }
 
+
+SEAL_C_FUNC Encryptor_EncryptReturnComponents(
+    void *thisptr, 
+    void *plaintext, 
+    bool disable_special_modulus, 
+    void *destination, 
+    void *u_destination, 
+    void *e_destination, 
+    void *pool_handle) 
```

Comment:
Do other functions take pool handles? I'd rather keep this detail internal if we can.